app/modules/code_provider/github/github_service.py

```
@@ -217,20 +266,35 @@ async def get_repos_for_user(self, user_id: str):
                 elif account_type == "Organization" and account_login in org_logins:
                     user_installations.append(installation)
 
+
             repos = []
             for installation in user_installations:
                 app_auth = auth.get_installation_auth(installation["id"])
                 github = Github(auth=app_auth)
```

Ruff (0.8.2)

Local variable `github` is assigned to but never used

Remove assignment to unused variable `github`

(F841)